drivers/adc-dac/ad5592r/ad5592r.c

```diff
@@ -3,7 +3,7 @@
  *   @brief  Implementation of AD5592R driver.
  *   @author Mircea Caprioru (mircea.caprioru@analog.com)
 ********************************************************************************
- * Copyright 2018(c) Analog Devices, Inc.
+ * Copyright 2018, 2020(c) Analog Devices, Inc.
```

Comment:
I have the same comment as in the other commit regarding the date.
This should be a separate commit changing all dates, otherwise suppose there is a commit revert due to a bug, and the reverse operation will reverse more than what you intend... :)